hrjobcontract/xml/length_of_service.xml

```diff
@@ -19,23 +19,5 @@
     </CustomGroup>
   </CustomGroups>
   <CustomFields>
-    <CustomField>
```

Comment:
Are you sure the requirement is remove the `custom_field`? I think this will remove rather than setting the field to view only.